vmas/simulator/sensors.py

```diff
@@ -6,12 +6,13 @@
 
 import typing
 from abc import ABC, abstractmethod
-from typing import Callable, List, Union
+from typing import Callable, List, Tuple, Union
 
 import torch
 
 import vmas.simulator.core
 import vmas.simulator.utils
+from vmas.simulator.utils import Color
```

Comment:
Have you checked this works? 

the reason the imports above are so general is because importing specific elements was causing me circular imports error